hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieCompactor.java

```diff
@@ -293,4 +297,17 @@ private String getSchemaFromLatestInstant() throws Exception {
     Schema schema = schemaUtil.getTableAvroSchema(false);
     return schema.toString();
   }
+
+  private int handleErrors(HoodieCommitMetadata metadata, String instantTime) {
+    List<HoodieWriteStat> writeStats = metadata.getPartitionToWriteStats().entrySet().stream().flatMap(e ->
+        e.getValue().stream()).collect(Collectors.toList());
+    long errorsCount = writeStats.stream().mapToLong(HoodieWriteStat::getTotalWriteErrors).sum();
+    if (errorsCount == 0) {
+      LOG.info(String.format("Table imported into hoodie with %s instant time.", instantTime));
```

Comment:
pls fix the log message,  we are doing compact not Import data to hoodie table

Reply:
Changed